package.json

```diff
@@ -27,13 +27,6 @@
 		"import": "./dist/index.mjs",
 		"types": "./dist/index.d.ts"
 	},
-	"imports": {
-		"#get-tsconfig": {
-			"types": "./src/index.ts",
-			"development": "./src/index.ts",
-			"default": "./dist/index.mjs"
-		}
-	},
```

Comment:
Why was this removed?

Reply:
yarn pnp seems does not support this yet.